Samples/Blinky/Blinky-samc21xpro.lpi

```diff
@@ -73,33 +73,31 @@
   <CompilerOptions>
     <Version Value="11"/>
     <Target>
-      <Filename Value="Blinky"/>
+      <Filename Value="bin/$(TargetCPU)-$(TargetOS)/samc21/$NameOnly($(ProjFile))"/>
     </Target>
     <SearchPaths>
       <IncludeFiles Value="../../Source;$(ProjOutDir)"/>
       <OtherUnitFiles Value="../../Source"/>
-      <UnitOutputDirectory Value="lib/$(TargetCPU)-$(TargetOS)"/>
+      <UnitOutputDirectory Value="lib/$(TargetCPU)-$(TargetOS)/samc21"/>
     </SearchPaths>
     <Parsing>
       <SyntaxOptions>
         <UseAnsiStrings Value="False"/>
       </SyntaxOptions>
     </Parsing>
     <CodeGeneration>
-      <HeapSize Value="8192"/>
-      <StackSize Value="8192"/>
       <TargetCPU Value="arm"/>
       <TargetOS Value="embedded"/>
     </CodeGeneration>
     <Linking>
       <Debugging>
+        <GenerateDebugInfo Value="False"/>
```

Comment:
Also here, any specific reason for this? Having Debug Info will not change the size of the deployed code, so why turn DebugInfo off?

Reply:
Again, not anything particular.
My normal practice is to try to get things running with as little extra settings as possible.